_layouts/player_example.html

```diff
@@ -0,0 +1,125 @@
+{% include head.html %}
+
+{% include main_nav.html %}
+
+<div class="container bs-docs-container">
+  <div class="row">
+
+    <div class="col-md-5">
+      <div class="docs-sidebar">
+        <div class="row">
+          <div class="col-md-6">
+            {% include left_nav.html %}
+          </div>
+
+          <div class="col-md-6">
+            {% include example_tab.html %}
+            <!-- </div> -->
+          </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-md-7" role="main">
+    <div class="bs-docs-section" markdown="1">
+
+      <h1>{{ page.description }} </h1>
+
+      {% if page.why_link %}
+        <h4>Who should use this example:</h4>
+
+        <ul>
+          <li>Please refer to the <a href="{{page.why_link}}" target="_blank">product feature launch notes</a>.</li>
+        </ul>
+      {% endif %}
+      <p>This page is an example of using prebid video with a specific player. For a general guide to using video in prebid, click <a href="prebid.org/dev-docs/show-video-with-a-dfp-video-tag.html">here.</a></p>
```

Comment:
Can you please change the 'prebid.org' in the link to use `{{site.github.url}}`?  The hardcoded URL is broken without the `http`.

Sorry for not mentioning that somewhere, I need to put together a contributing guide for the site to make it easier.  (P.S. This PR is awesome, sorry if I'm picking nits)